webapp/src/api/QuestionAnswerApi.ts

```diff
@@ -31,6 +31,21 @@ export const getAllQnA = async () => {
   }
 }
 
+export const getAllQnAForCourse = async (courseId: string) => {
```

Comment:
Let's try to use 'Exchanges' as a naming convention instead of 'Questions', 'QnA', etc.